app/setup/environment.js

```diff
@@ -102,7 +102,7 @@ exports.init = async function (app) {
     await next();
   });
 
-  const accessLogStream = fs.createWriteStream(`${__dirname}/../log/${env}.log`, { flags: 'a' });
+  const accessLogStream = fs.createWriteStream(`${__dirname}/../../log/${env}.log`, { flags: 'a' });
```

Comment:
would be great to have log directory also configurable

Reply:
Should we support relative paths here? Now the default is "./logs" relative to the root of the repository (not even cwd).

Reply:
@davidmz what I would normally do: get process working directory and use `${workdir}/logs` as the default for `logdir`.
And allow to override `logdir` via config/env.